src/content/docs/dropins/cart/cart-slots.mdx

```diff
@@ -175,9 +175,7 @@ The following example demonstrates how to extend the Thumbnail slot.
 
 The Footer slot allows you to add content to the bottom of the CartSummaryList container.
 
-#### Example: Extend the Footer slot
```

Comment:
Since you're removing this header and the one on line 216, should you also revisit line 112 (extend thumbnail example)? Maybe not, but the break in continuity caught my eye.